exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/vendors/MysqlDialect.kt

```diff
@@ -308,7 +318,10 @@ open class MysqlDialect : VendorDialect(dialectName, MysqlDataTypeProvider, Mysq
 
     override fun isAllowedAsColumnDefault(e: Expression<*>): Boolean {
         if (super.isAllowedAsColumnDefault(e)) return true
-        val acceptableDefaults = arrayOf("CURRENT_TIMESTAMP", "CURRENT_TIMESTAMP()", "NOW()", "CURRENT_TIMESTAMP(6)", "NOW(6)")
+        if ((currentDialect is MariaDBDialect && fullVersion >= "10.2.1") || (currentDialect !is MariaDBDialect && fullVersion >= "8.0.13")) {
```

Comment:
It could be better to override this function inside `MariaDBDialect` also insetead of checking for it here. Could be like:
```kotlin
override fun isAllowedAsColumnDefault(e: Expression<*>): Boolean {
    if (fullVersion >= "10.2.1") return true
    return super.isAllowedAsColumnDefault(e)
}
```